lib/url-decorator.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { CompositeDecorator } from 'draft-js';
+import urlRegex from 'url-regex';
+import normalizeUrl from 'normalize-url';
+import { omit } from 'lodash';
+import { viewExternalUrl } from './utils/url-utils';
+
+const urlRe = urlRegex();
+const markdownLinkRe = /\(|\]:\s$/m;
```

Comment:
there was actually a mistake in this pattern, I've fixed it now. let me know if the comment is clear enough